grandnode/grandnode2
# The widget slider is showing other stores' images.

Resolves #193 
Type: **bugfix**

## Issue
The widget slider is showing other stores' images.
When you have two stores in the same instance and add a widget slider to both the stores with limited stores field set to respective store for each slider
When you browse each store, it will show all slides add to the widget slider instead of filtering them based on the limited store's field values

## Solution
ACL service authorize method should check LimitedStores flag along with if entity consists of stores count

## Breaking changes
It will not break any other functionalities, it is an extra check 

## Testing
1. Create two stores
2. Enable widget slider from plugins to both stores
3. Add slides to the widget slider and distribute them to different stores
4. Browse the store and see the home page if these images are coming from the respective mapping store.
